source/reference/minio-mc-admin/mc-admin-prometheus.rst

```diff
@@ -56,7 +56,12 @@ Syntax
 
       The type of metrics to scrape.
 
-      Valid values are ``cluster``, ``node``, or ``bucket``.
+      Valid values are:
+
+      - ``bucket``
+      - ``cluster``
+      - ``node``
+      - ``resource``
```

Comment:
Maybe add the node included in cluster lines here as well?

Reply:
Ooo good idea 👍🏻